packages/ui-components/src/display/natValue/stringifyRatioAsPercent.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+
+import { assert, details } from '@agoric/assert';
+import { captureNum } from './helpers/captureNum';
+import { roundToDecimalPlaces } from './helpers/roundToDecimalPlaces';
+
+const PERCENT_BASE = 100n;
+
+/**
+ *
+ * @param {Ratio} ratio
+ * @param {object} options
+ * @param {number} options.numDecimalPlaces
+ * @param {number} options.denomDecimalPlaces
+ * @param {number} options.placesToShow
+ * @returns {string}
+ */
+export const stringifyRatioAsPercent = (ratio, options) => {
+  if (ratio === null || ratio === undefined) {
+    return '0';
+  }
+  assert(
+    ratio && ratio.numerator,
+    details`Ratio ${ratio} did not look like a ratio`,
+  );
+  assert(ratio);
+  const { numDecimalPlaces, denomDecimalPlaces, placesToShow = 2 } =
+    options || {};
+  assert(
+    options.numDecimalPlaces,
+    `numDecimalPlaces ${options.numDecimalPlaces} must be defined`,
+  );
+  assert(
+    options.denomDecimalPlaces,
+    `denomDecimalPlaces ${options.denomDecimalPlaces} must be defined`,
+  );
+  const denomPower = 10n ** BigInt(denomDecimalPlaces);
+  const numPower = 10n ** BigInt(numDecimalPlaces);
+  const numerator = ratio.numerator.value * denomPower * PERCENT_BASE;
```

Comment:
Rather than having hardwired 2 places of precision with "percent", should this be an option?

Reply:
hmm, good point.

Reply:
Hmm, I thought through passing the setting 'percent' or passing through `2` or `100`, but I think that gets confusing. I don't think we'll ever want a different setting if we actually want a percent, since all it's doing is multiplying by 100. So I think that intention makes the most sense as a method name.

Reply:
I am pretty sure we will want something here for 120% liquidations vs. 0.005% for fees (Joe estimated our fees need to be 0.5 to 5 basis points).  That's some combination of accuracy and places to show, so I'm not sure what to propose in the API.